gprofiler/metadata/application_metadata.py

```diff
@@ -4,16 +4,15 @@
 #
 
 import functools
-from subprocess import CompletedProcess
 from threading import Event, Lock
 from typing import Any, Dict, Optional
+from . import get_exe_version
```

Comment:
```suggestion
from gprofiler.metadata import get_exe_version
```
Prefer absolute imoports - that's our convention